{
  "code": 422,
  "data": [
    "Could not resolve related resources: RepositoryMaintainerDto(id=0, user=UserProjection(id=5, name=null, login=null, email=null), repository=RepositoryProjection(id=3333, name=null, publicationUri=null, published=false, technology=null), deleted=false, entity=null)"
  ],
  "messageCode": "error.validation",
  "message": "Validation failed for the resource.",
  "status": "ERROR"
}
